.ci/install_kata_image.sh

```diff
@@ -65,6 +65,8 @@ install_packaged_image() {
 		chronic sudo -E dnf install -y "$PACKAGED_IMAGE" || rc=1
 	elif [ "$ID"  == "centos" ]; then
 		chronic sudo -E yum install -y "$PACKAGED_IMAGE" || rc=1
+	elif [ "$ID" == "opensuse-leap" ]; then
```

Comment:
Are you using Leap 15 ooi? I've got a Leap 42.3 system, but that sets `ID=opensuse`.

/cc @marcov.

See also: https://github.com/kata-containers/tests/pull/1144

Reply:
@jodh-intel yes, Leap 15

Reply:
NAME="openSUSE Leap"
VERSION="15.0"
ID="opensuse-leap"
ID_LIKE="suse opensuse"
VERSION_ID="15.0"
PRETTY_NAME="openSUSE Leap 15.0"

Reply:
Hmm, since 42.3 is not yet EOL, I wonder if we want to do something like this then:

```sh
[[ "$ID" =~ ^opensuse.*$ ]]
```

I'll update kata-manager for that too...

Reply:
My kata-manager PR: https://github.com/kata-containers/tests/pull/1151